CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributing to Raygun4Android
+
+## Project and library organisation
+
+Building the project at this stage requires Android Studio 3.2+ and Java 8 or newer.
+
+The project consists of two modules:
+
+/app: A sample app demonstrating the use of Raygun4Android with a dependency to the "provider" module of the project.
+
+/provider: Android library project that contains the actual Raygun4Android provider.
+
+## Building
+
+### Building from Android Studio
+
+Setup the app or provider module in Run - Edit Configurations. Select the module in the UI and build by pressing the green triangle button next to it.
+
+Building the app module will also resolve the dependency to the provider module.
+
+build.gradle in the app module defines which version of the library is being used:
+
+````
+// Build from local provider library src
+implementation project(':provider')
+
+// Build from external repo
+//implementation 'com.raygun:raygun4android:4.0.0-alpha2'
```

Comment:
update to beta2?

Reply:
That's just a sample explaining the two different ways of building it with either the local or the remotely hosted library. Not a deal breaker for the merge imho, but yeah - would probably not harm to keep it up to date *or* more generic (so that we don't have to update that document all the time).

Reply:
Changed the wording in ff55520